### PR TITLE
DFPL-1414: Fix issue of Bulk Print receiving duplicated requests

### PR DESCRIPTION
**Pull request by qasimnawaz-moj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DFPL-1414

### Change description ###

Fix issue of Bulk Print receiving duplicated requests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/SendLetterService.java [hunk 1]
```diff
@@ -64,7 +64,10 @@ public List<SentDocument> send(DocumentReference mainDocument, List<Recipient> r
                 SendLetterResponse response = sendLetterApi.sendLetter(authTokenGenerator.generate(),
                     new LetterWithPdfsRequest(List.of(coverDocumentEncoded, mainDocumentEncoded),
                         SEND_LETTER_TYPE,
-                        Map.of("caseId", caseId, "documentName", mainDocument.getFilename())));
+                        Map.of(
+                            "caseId", caseId,
+                            "documentName", mainDocument.getFilename(),
+                            "recipients", recipients)));
```
Comment (DanCatchpole) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/SendLetterService.java:70]: Think we might want just the single recipient - looks like we send an individual letter (with unique cover letter) to each recipient

Comment (qasimnawaz-moj) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/SendLetterService.java]: Actually I think youre right, I pushed the changes

**Review by DanCatchpole — DISMISSED** [review]
lgtm!

**Review by so99y — APPROVED** [review]
lgtm

**Pull request merged**